packages/access-client/src/stores/types.ts

```diff
@@ -5,8 +5,8 @@ import {
   DelegationMeta,
   SpaceMeta,
 } from '../types.js'
-import { RSASigner } from '@ucanto/principal/rsa'
 import { SignerArchive, DID } from '@ucanto/interface'
+import { RSA } from '@ucanto/principal/.'
```

Comment:
Whaaat ? Is this same as

```suggestion
import { RSA } from '@ucanto/principal'
```

And if so why not use the above one ?